plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/extractor/PluginMetadata.java

```diff
@@ -37,6 +37,11 @@ public class PluginMetadata extends CacheEntry<PluginMetadata> implements Serial
      */
     private boolean hasJenkinsfile;
 
+    /**
+     * Minimum JDK version required by the plugin
+     */
+    private int jdkVersion;
```

Comment:
Why an int when we have now an Enum for the JDK model ?
Why not getting list of JDK version configured at let the call (PluginModernizer) use they JDK they need (minimum or maximum)